e2e-tests/portal/cypress/e2e/journeys/admin/create-user/create-a-user.spec.js

```diff
@@ -200,9 +204,10 @@ context('Admin user creates a new user', () => {
     });
 
     // as the string has object characters, need to use parseSpecialCharSequences
-    createUser.username().type(USER_WITH_INJECTION.username, { parseSpecialCharSequences: false });
-    createUser.firstname().type(USER_WITH_INJECTION.firstname);
-    createUser.surname().type(USER_WITH_INJECTION.surname);
+    createUser.username().type(USER_WITH_INJECTION.username, { delay: 0, parseSpecialCharSequences: false });
```

Comment:
could this be handled by keyboardInput?

Reply:
It could, but I don't think it should. `parseSpecialCharSequences` is only used here, it is very specific to this test and therefore IMO doesn't need to be in a generic/common command.